accessibility-checker-engine/src/v2/checker/accessibility/rules/rpt-marquee-rules.ts

```diff
@@ -28,10 +28,12 @@ let a11yRulesMarquee: Rule[] = [
         context: "dom:marquee",
         run: (context: RuleContext, options?: {}): RuleResult | RuleResult[] => {
             const ruleContext = context["dom"].node as Element;
+            //skip the rule
+            if (RPTUtil.isNodeHidden(ruleContext)) return null;
```

Comment:
This is a rule related to visual issue and not AT based, so shouldn't be ignored.

Reply:
removed